### ISSUE TITLE
MAGIC FLV test condition gives false negative, get bytes test str

### ISSUE THREAD

**Issue by shithead** [KnugiHK/rtmplite3#3]
In file flv.py line 57-62
```python
magic, version, flags, offset = struct.unpack(
    '!3sBBI', self.fp.read(9))
if self._debug:
    print('FLV.open() hdr=', magic, version, flags, offset)
if magic != 'FLV':
 raise ValueError('This is not a FLV file')
```

magic contains bytes see https://docs.python.org/3/library/struct.html#format-characters but in line 61 is test on 'FLV' string.

output from line 60: `FLV.open() hdr= b'FLV' 1 5 9`

**Comment by KnugiHK** [comment]
I probably missed this piece of code when I converting the project from Python 2 to Python 3. Thanks for your report!

**Comment by KnugiHK** [comment]
I will leave this open until a new release is made.

**Comment by KnugiHK** [comment]
Released in 0.4.0. I am closing this issue.
